auth_jwt/models/auth_jwt_validator.py

```diff
@@ -292,11 +305,7 @@ def unlink(self):
         return super().unlink()
 
     def _get_jwt_cookie_secret(self):
-        secret = self.env["ir.config_parameter"].sudo().get_param("database.secret")
-        if not secret:
-            _logger.error("database.secret system parameter is not set.")
-            raise ConfigurationError()
-        return secret
+        return self.secret_key
```

Comment:
If the database secret is not adequate for your use case we could a new secret dedicated to signing/encrypting the cookie on the jwt validator record?

Reply:
Modified encode and decode functions to use the fields renew_cookie_secret and renew_cookie_algorithm